types/defines/workflows.d.ts

```diff
@@ -0,0 +1,8 @@
+
+/**
+ * NonRetryableError allows for a Workflow to throw a "fatal" error as in,
+ * an error that makes the instance fail immediately without triggering a retry.
+ */
+export class NonRetryableError extends Error {
+  public constructor(message: string, name?: string);
```

Comment:
Are you intentionally omitting the `__brand` string from the types? If so, would be good to document the reason so someone doesn't come along later and add it.

Reply:
done